src/aufgabe1.java

```diff
@@ -0,0 +1,35 @@
+import java.lang.reflect.Array;
+import java.util.Scanner;
+public class aufgabe1 {
+
+   public static void main(String[] args) {
+
+      aufgabe_1();
+      aufgabe_2();
+   }
+
+   private static void aufgabe_1(){
+      Scanner scan = new Scanner(System.in);
+      System.out.print("Write a double number: ");
+      double x = scan.nextDouble();
+
+      for (int i = 1; i < 9; i++) {
+         double div = x % i;
+         if (div == 0.0){
+            System.out.println("Mod:" + i);
+         }
+      }
+   }
+
+   private static void aufgabe_2(){
+      String text = "gbcFarnxl";
+      int schift = 13;
+      System.out.print("Word is: ");
+      for (int i = 0; i < text.length(); i++) {
+         char character = text.charAt(i);
+         int ch;
+         ch = (int) character + schift;
```

Comment:
That is not completely correct. The `char` representation is not exactly our alphabet. Therefore, if you add 13 to `'n'` you will get a` '{'.` Have a look [here](https://upload.wikimedia.org/wikipedia/commons/thumb/1/1b/ASCII-Table-wide.svg/1280px-ASCII-Table-wide.svg.png) what the range for A-Z and a-z is. (_Hint:_ you will need % again)